style.css

```diff
@@ -51,25 +53,26 @@ img {
 
 p {
     line-height: 1.5em;
-background-color: #A9E7BF;
 }
 
-.copy{
-background-color: #FFF;
+.copy {
+    background-color: #FFF;
 }
 
-.container.textbox{
-background-color: #FFF;
-padding: 5px;
+.container.textbox {
+    background-color: #FFF;
+    padding: 10px;
+    margin: 10px auto;
```

Comment:
adds a top and bottom margin of 10px to create negative space around image sections